### PR TITLE
Enhancement: Roll Labeling

### PR DESCRIPTION
**Pull request by TheRayOfSeasons**
This enhances the roll system by allowing the user to label their rolls.

### CODE REVIEW

File: bot/modules/roll/commands/roll.js [hunk 1]
```diff
@@ -30,11 +35,29 @@ module.exports = class extends Command
     if(!validFormat.test(text))
       return context.chat('Invalid format.');
 
+    let label = rawText.replace(rollFormatRegex, '');
+    const labelRegexValue = label
+      .toLowerCase()
+      .split('')
+      .map(letter => {
+        const escapables = [
+          '.', '^', '$', '*', '+', '?', '(', ')', '[', '{', '\\', '|'
```
Comment (esfox) [inline bot/modules/roll/commands/roll.js:44]: Baka pwede 'to 1 line pero kung mas okay ka dito oks lang.

Comment (TheRayOfSeasons) [reply bot/modules/roll/commands/roll.js:44]: Mag exceed to sa 100 lines pag naka 1 line eh.

File: bot/modules/roll/commands/roll.js [hunk 2]
```diff
@@ -94,7 +117,8 @@ module.exports = class extends Command
     let total = eval(operation);
     total = Math.round(total);
 
-    context.chat(`Breakdown: ${breakdown}\nTotal: ${total}`);
+    label = hasText.test(label) ? label : 'Result';
+    context.chat(`Breakdown: ${breakdown}\n**${label}**: ${total}`);
```
Comment (esfox) [inline bot/modules/roll/commands/roll.js:121]: Try mo gamitin `context.send(tittle, description)`.
Ginagawa nito nagsesend ng embed parang ganito...
image

'Yong nakabold text sa pic ("Wheel of TWICE"), 'yon 'yong first param `title`...
'Yong other text ("You chose Jihyo..."), 'yon 'yong second param `description`. Pwede Discord markdown dito.

Suggestion lang naman 'to.

Comment (TheRayOfSeasons) [reply bot/modules/roll/commands/roll.js:121]: Ayt ayt

Comment (TheRayOfSeasons) [reply bot/modules/roll/commands/roll.js]: Resolved this as `context.chat()` to keep it quotable.

**Pull request merged**